openvdb/python/pyOpenVDBModule.cc

```diff
@@ -229,7 +229,7 @@ struct MatConverter
             for (int j = 0; j < MatT::size; ++j) { rowObj.append(m(i, j)); }
             obj.append(rowObj);
         }
-        return obj;
+        return std::move(obj);
```

Comment:
```
python/pyOpenVDBModule.cc:232:16: warning: local variable 'obj' will be copied despite being returned by name [-Wreturn-std-move]
        return obj;
python/pyOpenVDBModule.cc:232:16: note: call 'std::move' explicitly to avoid copying
```